README.md

```diff
@@ -1,7 +1,9 @@
-This guide is a collection of techniques for improving the security and privacy of a modern Apple Macintosh computer ("MacBook") running a recent version of macOS (formerly known as "OS X").
+This guide is a collection of techniques for improving the security and privacy of [Apple silicon](https://support.apple.com/116943) Mac computers running a recent version of macOS. **Using older Macs leaves you open to [security vulnerabilities](https://github.com/axi0mX/ipwndfu?tab%253Dreadme-ov-file#checkm8) on the hardware level that Apple can't patch**. Apple silicon Macs are the minimum recommendation but as a general rule, newer chips are always more secure.
```

Comment:
Can we somehow qualify what "older Macs" refers to? AFAIK Apple supports the latest two OS releases, but is it clear what hardware is updated?

Reply:
Clarified Intel Macs.